src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerPanel.java

```diff
@@ -746,6 +749,7 @@ private void initializePanelItems() {
 		subgraphPanel.setLayout(new BoxLayout(subgraphPanel, BoxLayout.PAGE_AXIS));
 		TitledBorder subgraphBorder = BorderFactory.createTitledBorder("Output");
 		subgraphPanel.setBorder(subgraphBorder);
+		subgraphPanel.add(_includePathScoreTiesOption);
```

Comment:
Move the checkbox option below k (# of paths) text field in the Algorithm section